sky/global_user_state.py

```diff
@@ -171,11 +180,19 @@ def update_last_use(cluster_name: str):
 
 def remove_cluster(cluster_name: str, terminate: bool):
     """Removes cluster_name mapping."""
+    handle = get_handle_from_cluster_name(cluster_name)
+    metadata = get_cluster_metadata(cluster_name)
+
+    if handle and metadata:
+        start_time = get_cluster_launch_time(cluster_name)
+        end_time = int(time.time())
```

Comment:
why do we need to convert it to `int`?

Reply:
default is float, and int is seconds for time, which should be accurate enough and can save on space compared to storing floats

Reply:
Good point! Sounds good to me.